frontend/src/metabase/query_builder/components/filters/FilterPopover/FilterPopoverHeader.tsx

```diff
@@ -54,8 +54,9 @@ export default function FilterPopoverHeader({
             pr2: !showOperatorSelectorOnOwnRow,
           })}
           title={
-            (field.table ? field.table.displayName() + " – " : "") +
-            field.displayName()
+            (field.table?.displayName()
+              ? field.table.displayName() + " – "
+              : "") + field.displayName()
```

Comment:
There is probably a more beautiful and idiomatic way to write this in JS.
But I wanted to first check if this is even acceptable idea?

Reply:
This looks fine to me. Although, I'm more curious about the expected behavior. From your screenshot
<img src="https://user-images.githubusercontent.com/31325167/206707915-e0d8d9eb-ecce-4611-bd92-4892db80130b.png" />

`Count` is a column on an aggregated result. I believe it's correct that we don't say `Count` belongs to any table, so that's why we left out the name right? Because if you filter before the summarize step all fields will be in this format `table_name – field_name`, since we can infer which fields belong to which table.

Reply:
We shouldn't have logic like this on the FE. It seems that we should call a proper `metabase-lib` method, like 
```
field.displayName({ includeTable: true }) 
```

Reply:
@nemanjaglumac could you check if that works for this case?

Reply:
`field.displayName({ includeTable: true })` in this case returns `→ Count`.
So, we're still missing the `table.displayName` but printing the `→`

What would be the correct way to proceed? @ranquild 
Does this method from the `metabase-lib` need to be updated?

Reply:
@kulyk could you take a look at this thread please?